reddit_liveupdate/controllers.py

```diff
@@ -451,18 +452,62 @@ def GET_about(self):
     )
     def GET_discussions(self, num, after, reverse, count):
         """Get a list of reddit submissions linking to this thread."""
-        builder = url_links_builder(
-            url="/live/" + c.liveupdate_event._id,
-            num=num,
-            after=after,
-            reverse=reverse,
-            count=count,
-        )
+
+        show_hidden = c.liveupdate_permissions.allow("discussions")
+        builder = get_discussions(
+            c.liveupdate_event, limit=50, show_hidden=show_hidden)
         listing = LinkListing(builder).listing()
+        listing.render_class = pages.LiveUpdateDiscussionsListing
+        listing.submit_url = pages.make_submit_url(c.liveupdate_event)
         return pages.LiveUpdateEventPage(
             content=listing,
         ).render()
 
+    @require_oauth2_scope("livemanage")
+    @validatedForm(
+        VLiveUpdateContributorWithPermission("discussions"),
+        VModhash(),
+        link=VLink("link"),
+    )
+    @api_doc(
+        section=api_section.live,
+    )
+    def POST_hide_discussion(self, form, jquery, link):
+        """Hide a linked comment thread from the discussions sidebar and listing.
+
+        Requires the `discussions` permission for this thread.
+
+        See also: [/api/live/*thread*/unhide_discussion](#POST_api_live_{thread}_unhide_discussion).
+
+        """
+        url = pages.make_event_url(c.liveupdate_event._id)
+        if link.url != url:
```

Comment:
Probably not very concerning, but does `VLink()` do implicit permission checks? If not you could test if fullnames for links in private SRs point to the livethread.

Reply:
Fixed.